client/lxd.go

```diff
@@ -272,3 +323,9 @@ func (r *ProtocolLXD) websocket(path string) (*websocket.Conn, error) {
 
 	return r.rawWebsocket(url)
 }
+
+func (r *ProtocolLXD) setupBakeryClient() {
+	r.bakeryClient = httpbakery.NewClient()
+	r.bakeryClient.Client = r.http
+	r.bakeryClient.AddInteractor(form.Interactor{Filler: schemaform.IOFiller{}})
```

Comment:
Sounds like this is going to assume that whatever is using our client library has a valid stdin/stdout for interactive handling, that's not guaranteed to be the case at all.

Reply:
yeah, my plan is to add support for non-interactive login (passing `--macaroon-form` parameters as a next step

Reply:
Feels like this part should be handled outside of the client/ codebase and we should just allow passing an Interactor through to the client package.

That way it's up to whatever uses our client package to decide what the right interactor is for them.